src/components/SquareBox/SquareBox.tsx

```diff
@@ -28,24 +27,23 @@ const styles = StyleSheet.create({
     marginRight: '15px',
   },
 });
-
+interface Shape {
+  color: string;
+  shape: string;
+}
 export function SquareBox() {
-  interface Shape {
-    color: string;
-    shape: string;
-  }
   const id = React.useId();
   const [squares, setSquares] = React.useState<Array<Shape>>([]);
   const addCircle = () => {
-    setSquares((squares) => ([
+    setSquares(() => ([
       ...squares, { color: randomElement(Object.values(COLORS)), shape: 'circle' },
     ]
     ));
   };
 
   const deleteSquare = () => {
     squares.pop();
-    setSquares((squares) => [...squares]);
```

Comment:
in this and in line 38 squares are not required but what to do with line 50 how should i change it ?